### DISCUSSION TITLE
Is there an example of SSR with react-navigation?

### DISCUSSION THREAD

**Discussion by UberMC** [necolas/react-native-web#2083] [Help] [answered]
Hey,

Does anyone have example code of react-native-web using server side rendering (SSR) and react-navigation.

**Comment by necolas** [comment, marked as answer]
See [React Navigation: Server Rendering](https://reactnavigation.org/docs/server-rendering)

**Comment by otim** [comment]
@UberMC did you manage to get this to work? The react navigation documentation lacks quite some important details to get this to work in my opinion. What configs are needed, what compiler has to be used, what command can be used for serving a local dev variant, how can it be built for production?
I saw you created a [boilerplate repo](https://github.com/UberMC/react-native-web-react-navigation-ssr), thank you for this!
However, when I try to replicate the same logic in my build, the build is failing because of some MODULE_NOT_FOUND errors. I _think_ the reason is that the import resolution from react-native to react-native-web is not working correctly.
Have you or has anyone else figured this out?
